tex/gregoriotex-spaces.tex

```diff
@@ -384,6 +384,15 @@
 
 % dimen keeping the shift computed with next function
 \newdimen\gre@dimen@eolshift
+\newcount\gre@count@protrusion@hyphen@eol %
+
+% protrusion for hyphens inerted by Lua
+\gre@count@protrusion@hyphen@eol = 50
```

Comment:
I'll add it for the sake of safety, but here (not in a macro), we're not in an environment where we have to avoid end of lines, and there is an end of line after the number, so TeX will see a `number` token, and then a `end of line` token, so it won't search for what's after the number. In an environment where we avoid end of lines, TeX would keep looking for what's after the number, which can lead to errors